bundles/AdminBundle/src/Controller/Admin/TranslationController.php

```diff
@@ -762,13 +761,6 @@ public function contentExportJobsAction(Request $request): JsonResponse
         $elements = array_values($elements);
 
         $elementsPerJob = 10;
-        if ($type == 'word') {
-            // the word export can only handle one document per request
-            // the problem is Document\Service::render(), ... in the action can be a $this->redirect() or exit;
-            // nobody knows what's happening in an action ;-) So we need to isolate them in isolated processes
-            // so that the export doesn't stop completely after a "redirect" or any other unexpected behavior of an action
-            $elementsPerJob = 1;
```

Comment:
Yes, I thought about leaving it in the core as well, but I was not satisfied to leave functionality in the core that is not used, when the bundles are not installed so I decided to move it out to the bundle, but we can still keep in the core but yes we would need a new param for the `elements_per_job`

Reply:
I saw that you added the `checkPermission` to your Controller, maybe I should do that too, since it should not work without the permissions? 
We do have to consider this for every bundle right?

Reply:
I will wait for the xliff changes to be merged and then adapt the changes corresponding to the `contentExportJobsAction`